server/rest/src/main/java/org/infinispan/rest/resources/ServerResource.java

```diff
@@ -91,15 +107,132 @@ public Invocations getInvocations() {
       DefaultCacheManager cacheManager = invocationHelper.getServer().getCacheManager(cacheManagerName);
 
       if (cacheManager == null) return notFoundResponseFuture();
-      CacheIgnoreManager ignoreManager = invocationHelper.getServer().getIgnoreManager(cacheManagerName);
-      Set<String> ignored = ignoreManager.getIgnoredCaches();
+      ServerStateManager serverStateManager = invocationHelper.getServer().getServerStateManager(cacheManagerName);
+      Set<String> ignored = serverStateManager.getIgnoredCaches();
       return asJsonResponseFuture(Json.make(ignored));
    }
 
    private CompletionStage<RestResponse> cacheManagers(RestRequest restRequest) {
       return asJsonResponseFuture(Json.make(invocationHelper.getServer().cacheManagerNames()));
    }
 
+   private CompletionStage<RestResponse> connectorStartStop(RestRequest restRequest) {
+      NettyRestResponse.Builder builder = new NettyRestResponse.Builder().status(NO_CONTENT);
+      String connectorName = restRequest.variables().get("connector");
+
+      ProtocolServer connector = invocationHelper.getServer().getProtocolServers().get(connectorName);
+      if (connector == null) return completedFuture(builder.status(NOT_FOUND).build());
+      if (connector.getTransport() == null) return completedFuture(builder.status(BAD_REQUEST).entity("Connector has no transport").build());
+
+      String cacheManagerName = invocationHelper.getServer().cacheManagerNames().iterator().next();
+      ServerStateManager serverStateManager = invocationHelper.getServer().getServerStateManager(cacheManagerName);
+
+      switch (restRequest.getAction()) {
+         case "start":
+            return serverStateManager.connectorStart(connectorName).thenApply(r -> builder.build());
+         case "stop":
+            return serverStateManager.connectorStop(connectorName).thenApply(r -> builder.build());
+      }
+      return completedFuture(builder.status(BAD_REQUEST).build());
+   }
+
+   private CompletionStage<RestResponse> connectorStatus(RestRequest restRequest) {
+      NettyRestResponse.Builder builder = new NettyRestResponse.Builder();
+      String connectorName = restRequest.variables().get("connector");
+
+      ProtocolServer connector = getProtocolServer(restRequest);
```

Comment:
I think we should add authz as part of the REST invocation framework so that we can declare required permissions and avoid checking them in the code. /cc @gustavonalle